src/plotman/log_parser.py

```diff
@@ -0,0 +1,140 @@
+        return m != None
+
+    # Time for phase 3 = 14537.188 seconds. CPU (82.730%) Mon Apr  5 05:51:11 2021
+    def phase3_duration(self, line: str, entry: PlotInfo) -> bool:
+        m = re.search(r"^Time for phase 3 = (\d+\.\d+) seconds", line)
+        if m:
+            entry.phase3_duration_raw = float(m.group(1))
+        return m != None
+
+    # Time for phase 4 = 924.288 seconds. CPU (86.810%) Mon Apr  5 06:06:35 2021
+    def phase4_duration(self, line: str, entry: PlotInfo) -> bool:
+        m = re.search(r"^Time for phase 4 = (\d+\.\d+) seconds", line)
+        if m:
+            entry.phase4_duration_raw = float(m.group(1))
+        return m != None
+
+    # Total time = 39945.080 seconds. CPU (123.100%) Mon Apr  5 06:06:35 2021
+    def total_time(self, line: str, entry: PlotInfo) -> bool:
+        m = re.search(r"^Total time = (\d+\.\d+) seconds", line)
+        if m:
+            entry.total_time_raw = float(m.group(1))
+        return m != None
+
+    # Copy time = 501.696 seconds. CPU (23.860%) Sun May  9 22:52:41 2021
+    def copy_time(self, line: str, entry: PlotInfo) -> bool:
+        m = re.search(r"^Copy time = (\d+\.\d+) seconds", line)
+        if m:
+            entry.copy_time_raw = float(m.group(1))
+        return m != None
+
+    # Starting plotting progress into temporary dirs: /farm/yards/901 and /farm/yards/901
+    def plot_dirs(self, line: str, entry: PlotInfo) -> bool:
+        m = re.search(r"^Starting plotting progress into temporary dirs: (.+) and (.+)$", line)
+        if m:
+            entry.tmp_dir1 = m.group(1)
+            entry.tmp_dir2 = m.group(2)
+        return m != None
+
+    # Using 4 threads of stripe size 65536
+    def threads(self, line: str, entry: PlotInfo) -> bool:
+        m = re.search(r"^Using (\d+) threads of stripe size (\d+)", line)
+        if m:
+            entry.threads = int(m.group(1))
+        return m != None
+
+    # "^Using (\\d+) buckets"
+    def buckets(self, line: str, entry: PlotInfo) -> bool:
+        m = re.search(r"^Using (\d+) buckets", line)
+        if m:
+            entry.buckets = int(m.group(1))
+        return m != None
+
+    # Buffer size is: 4000MiB
+    def buffer_size(self, line: str, entry: PlotInfo) -> bool:
+        m = re.search(r"^Buffer size is: (\d+)MiB", line)
+        if m:
+            entry.buffer = int(m.group(1))
+        return m != None
+
+    # Plot size is: 32
+    def plot_size(self, line: str, entry: PlotInfo) -> bool:
+        m = re.search(r'^Plot size is: (\d+)', line)
+        if m:
+            entry.plot_size = int(m.group(1))
+        return m != None
+
+    # Starting phase 1/4: Forward Propagation into tmp files... Sun May  9 17:36:12 2021
+    def plot_start_date(self, line: str, entry: PlotInfo) -> bool:
+        m = re.search(r'^Starting phase 1/4: Forward Propagation into tmp files\.\.\. (.+)', line)
+        if m:
+            entry.started_at = m.group(1)
```

Comment:
Couldn't we parse this here to be a `pendulum.datetime.DateTime` or such?  If so, note the type hint and default below as well.

https://github.com/ericaltendorf/plotman/blob/7df92df56ea6e99579bfbb83e4a55687c1380a39/src/plotman/job.py#L41-L43